sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -121,6 +123,23 @@ def process_encoded(self, encoded_windowed_values):
       self.output(decoded_value)
 
 
+class IterableState(object):
```

Comment:
The naming is a little weird that both `IterableState` and `*RuntimeState` (defined at the bottom of this file) are both used at runtime.  I don't have any suggestions, but maybe there is better naming?

Reply:
Yeah, renamed.